app/models/user.rb

```diff
@@ -171,6 +172,10 @@ def combined_search_terms
     I18n.transliterate([last_name, email, birth_name, phone_number_formatted, first_name].compact.join(" "))
   end
 
+  def webhook_endpoints
+    WebhookEndpoint.where(organisation_id: organisations.pluck(:id))
```

Comment:
Je pinaille sur le code, mais là, on peut faire:
```suggestion
    WebhookEndpoint.where(organisation_id: organisations.ids)
```
qui serait strictement équivalent, ou aussi:
```suggestion
    WebhookEndpoint.where(organisation: organisations)
```
qui a l’avantage de ne faire qu’un seul aller-retour en base de donnée, avec une subquery.

(Si on veut pousser encore plus loin, on peut même faire
```suggestion
    WebhookEndpoint.joins(:organisation).merge(u.organisations)
```
qui fait un `inner join` au lieu d’une subquery, mais là, on pinaille _vraiment_.)